### ISSUE TITLE
epochs of ResNet26-BN on CIFAR10

### ISSUE THREAD

**Issue by SakurajimaMaiii** [mr-eggplant/EATA#1]
Thanks for releasing code.
Sorry to bother you.
Could you provide epochs on training resnet26 on CIFAR10 training set?
Specically, 75 epochs or 150 epochs?
(I know use TTT's code)

**Comment by mr-eggplant** [comment]
Hi, thanks for your interest!
We train ResNet-26 on CIFAR-10 totally of 75 epochs: initial LR is 0.1 with MultiStepLR(optimizer, [50, 65]).

**Comment by SakurajimaMaiii** [comment]
Do you run it with different seeds?
I use TTT'code and train 75epochs, lr=0.1, lr decay at 50/65 epoch.
I got 8.16 error rate on clean test set
31.54 error rate on CIFAR10-C (average of 75,including 15 types and 5 levels) (28.4 in paper Table 4)
I just want to know whether this is normal....
I use this commond (in TTT'code)
![image](https://user-images.githubusercontent.com/83657651/181203258-3b4c9731-1dc2-4736-8936-0f2505c7de3b.png)

Or, coulde you share pretrain model on CIFAR-10 of ResNet26 ?

**Comment by mr-eggplant** [comment]
When measuring the 31.54 error rate on CIFAR10-C (average of 75 types), do you use the BN statistics (mean and variance) of training or testing data? (model.train() or model.eval(), I remember that I used 'model.train()', but not very sure.)
[current_ckpt.pth.zip](https://github.com/mr-eggplant/EATA/files/9197081/current_ckpt.pth.zip)

The pre-trained model is as above: statedict['net']

**Comment by SakurajimaMaiii** [comment]
yeah, set model.train() gets better results (almost the same as the results of the paper)
Thanks for your help!
